src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { LogTooltips } from 'src/workflows-app/utils/task-log-utils';
+
+type TroubleshootingBoxProps = {
```

Comment:
```suggestion
interface TroubleshootingBoxProps = {
```

> That said, we recommend you use interfaces over type aliases. Specifically, because you will get better error messages.
https://www.typescriptlang.org/play/?#example/types-vs-interfaces